### PR TITLE
Toyota: add LTA-related messages and more

### PR DESCRIPTION
**Pull request by sshane**
Found stuff like wheel nag while using LTA, a low freq copy of `STEER_TORQUE_SENSOR->STEER_ANGLE` higher freq turn signals, LKA active in another message, and deciphering more in this PR

However, it's unclear if any of these will be useful for controlling the car since they are all very low rate, with the highest being 5 Hz. Possible we can change some configuration though!

image

CLEAR_HDANDS_ON_ALERT vs torque:

image

image

image

### CODE REVIEW

**Comment by sshane** [comment]
somehow the camera publishes the gas pedal percent in one of these messages, not on a RAV4 Hybrid 2023 or Corolla 2019

image

File: toyota_nodsu_pt_generated.dbc [hunk 1]
```diff
@@ -180,61 +155,89 @@ BO_ 705 GAS_PEDAL: 8 XXX
...
- SG_ RELEASE_STANDSTILL : 31|1@0+ (1,0) [0|1] "" HCU
- SG_ ITS_CONNECT_LEAD : 39|8@0+ (1,0) [0|1] "" Vector__XXX
+ SG_ PERMIT_BRAKING : 30|1@0+ (1,0) [0|1] "" XXX
+ SG_ RELEASE_STANDSTILL : 31|1@0+ (1,0) [0|1] "" XXX
+ SG_ ITS_CONNECT_LEAD : 39|8@0+ (1,0) [0|1] "" XXX
  SG_ ACCEL_CMD_ALT : 47|8@0- (0.05,0) [0|0] "m/s^2" XXX
  SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX
 
-BO_ 836 PRE_COLLISION_2: 8 DSU
+BO_ 836 PRE_COLLISION_2: 8 XXX
```
Comment (sshane) [inline toyota_nodsu_pt_generated.dbc:192]: @deanlee is cabana supposed to preserve these ECU notations?

Comment (deanlee) [reply toyota_nodsu_pt_generated.dbc:192]: yes it should be kept, but it's all ignored now. I'll  submit a pr to fix this

**Comment by sshane** [comment]
only thing this PR has now is `LTA_RELATED`, closing for now

**Pull request closed without merging**